frontend/src/metabase/visualizations/components/settings/ChartSettingMaxCategories.tsx

```diff
@@ -33,10 +33,10 @@ export const ChartSettingMaxCategories = ({
   );
 
   const handleAggregationFunctionChange = useCallback(
-    (value: string | null) => {
+    (value: AggregationFunction) => {
```

Comment:
Nit: I find the name for this type confusing since it's not a function. Maybe it should be `AggregationFunctionName` or even just `AggregationName`?